client/browser/scripts/release-firefox.sh

```diff
@@ -8,7 +8,8 @@ rm -rf build/web-ext
 mkdir -p build/web-ext
 
 # Sign the bundle
-web-ext sign --source-dir ./build/firefox --artifacts-dir ./build/web-ext --api-key "$FIREFOX_AMO_ISSUER" --api-secret "$FIREFOX_AMO_SECRET"
+yarn add web-ext
```

Comment:
@philipp-spiess, do you have better ideas than installing `web-ext` here?
Adding it to the project's dev dependencies (https://github.com/sourcegraph/sourcegraph/pull/41655/commits/479fbc9817b5fc53abc6de25b5f849df14e84269) results in `Usage Error: Couldn't find a script named "web-ext"` ([buildkite](https://buildkite.com/sourcegraph/sourcegraph/builds/172569#01833cbd-d942-4998-a06f-8e3853b70b2d/152-220)).

Reply:
By the way, do you have an idea why it was failing?

Reply:
Hmm maybe it's just not available in the global `PATH`? You could try if this works instead:

```suggestion
./node_modules/.bin/web-ext sign --source-dir ./build/firefox --artifacts-dir ./build/web-ext --api-key "$FIREFOX_AMO_ISSUER" --api-secret "$FIREFOX_AMO_SECRET"
```

Reply:
Unfortunately, it still errors out
```
./scripts/release-firefox.sh: line 11: ./node_modules/.bin/web-ext: No such file or directory
```